shell/platform/embedder/embedder.h

```diff
@@ -861,6 +861,64 @@ typedef struct {
   const char* variant_code;
 } FlutterLocale;
 
+typedef int64_t FlutterEngineDartPort;
+
+typedef enum {
+  kFlutterEngineDartObjectTypeNull,
+  kFlutterEngineDartObjectTypeBool,
+  kFlutterEngineDartObjectTypeInt32,
+  kFlutterEngineDartObjectTypeInt64,
+  kFlutterEngineDartObjectTypeDouble,
+  kFlutterEngineDartObjectTypeString,
+  kFlutterEngineDartObjectTypeBuffer,
```

Comment:
Perhaps note that for this case Dart code will receive an instance of Uint8List.

Reply:
Done.